### PR TITLE
src: adjust monitor

### PR DESCRIPTION
**Pull request by hustjieke**

### CODE REVIEW

File: src/monitor/monitor.go [hunk 1]
```diff
@@ -9,17 +9,18 @@
 package monitor
 
 import (
-	"fmt"
+	"config"
```
Comment (andyli029) [inline src/monitor/monitor.go]: move down

File: src/monitor/monitor.go [hunk 2]
```diff
@@ -9,17 +9,18 @@
...
+	"github.com/xelabs/go-mysqlstack/xlog"
 )
 
 var (
-	webMonitorPort = "13308"
-	webMonitorAddr = "0.0.0.0"
-	webMonitorURL  = "/metrics"
+	webMonitorPort string
+	webMonitorAddr string
+	webMonitorURL  string
```
Comment (andyli029) [inline src/monitor/monitor.go]: delete the three vars

File: src/monitor/monitor.go (hunk 1 above)
Comment (hustjieke) [reply src/monitor/monitor.go]: thks, alread done!

File: src/monitor/monitor.go (hunk 2 above)
Comment (hustjieke) [reply src/monitor/monitor.go]: done!

**Pull request closed without merging**